reference/conanfile.rst

```diff
@@ -0,0 +1,38 @@
+.. _conanfile_reference:
+
+conanfile.py
+============
+
+
+.. currentmodule:: conan
+.. autoclass:: ConanFile
+
+   .. code-block:: python
+
+       from conan import ConanFile
+
+       class HelloConan(ConanFile):
+           ...
```

Comment:
This looks a bit weird to me as it is, I would give a bit of introduction or maybe remove it? Is the autoclass really helpful there?
![image](https://user-images.githubusercontent.com/5045666/170232007-e9383723-87ee-4b34-b96a-e37f6228b42f.png)